resources/views/modals/redo-eula.php

```diff
@@ -0,0 +1,21 @@
+<!-- Modal -->
+<div class="modal fade" id="redoEula" tabindex="-1" role="dialog" aria-labelledby="redoEulaLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">Send...</div>
+                <div class="modal-body">
+                    <h5>Remind User to Sign</h5>
```

Comment:
This would need a form tag here - it performs an action, and all interactions that cause data to change or cause an event to happen need to be in a POST, not a link or a GET, FYI :)